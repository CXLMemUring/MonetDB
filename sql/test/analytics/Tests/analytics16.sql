start transaction;
create table analytics (aa int, bb int);
insert into analytics values (15, 3), (3, 1), (2, 1), (5, 3), (NULL, 2), (3, 2), (4, 1), (6, 3), (8, 2), (NULL, 4);

select covar_samp(aa, aa) from analytics;
select covar_samp(bb, bb) from analytics;
select covar_samp(aa, bb) from analytics;
select covar_samp(bb, aa) from analytics;

select covar_pop(aa, aa) from analytics;
select covar_pop(bb, bb) from analytics;
select covar_pop(aa, bb) from analytics;
select covar_pop(bb, aa) from analytics;


select covar_samp(aa, aa) from analytics group by aa;
select covar_samp(bb, bb) from analytics group by aa;
select covar_samp(aa, bb) from analytics group by aa;
select covar_samp(bb, aa) from analytics group by aa;

select covar_pop(aa, aa) from analytics group by aa;
select covar_pop(bb, bb) from analytics group by aa;
select covar_pop(aa, bb) from analytics group by aa;
select covar_pop(bb, aa) from analytics group by aa;


select covar_samp(aa, aa) from analytics group by bb;
select covar_samp(bb, bb) from analytics group by bb;
select covar_samp(aa, bb) from analytics group by bb;
select covar_samp(bb, aa) from analytics group by bb;

select covar_pop(aa, aa) from analytics group by bb;
select covar_pop(bb, bb) from analytics group by bb;
select covar_pop(aa, bb) from analytics group by bb;
select covar_pop(bb, aa) from analytics group by bb;


select covar_samp(aa, aa) from analytics group by aa, bb;
select covar_samp(bb, bb) from analytics group by aa, bb;
select covar_samp(aa, bb) from analytics group by aa, bb;
select covar_samp(bb, aa) from analytics group by aa, bb;

select covar_pop(aa, aa) from analytics group by aa, bb;
select covar_pop(bb, bb) from analytics group by aa, bb;
select covar_pop(aa, bb) from analytics group by aa, bb;
select covar_pop(bb, aa) from analytics group by aa, bb;



select covar_samp(aa, 1) from analytics group by aa;
select covar_pop(aa, 1) from analytics group by aa;
select covar_samp(aa, 1) from analytics group by bb;
select covar_pop(aa, 1) from analytics group by bb;
select covar_samp(aa, 1) from analytics group by aa, bb;
select covar_pop(bb, 1) from analytics group by aa, bb;

select covar_samp(1, aa) from analytics group by aa;
select covar_pop(1, aa) from analytics group by aa;
select covar_samp(1, aa) from analytics group by bb;
select covar_pop(1, aa) from analytics group by bb;
select covar_samp(1, aa) from analytics group by aa, bb;
select covar_pop(1, aa) from analytics group by aa, bb;

select covar_samp(1, 1) from analytics group by aa;
select covar_pop(1, 1) from analytics group by aa;
select covar_samp(1, 1) from analytics group by bb;
select covar_pop(1, 1) from analytics group by bb;
select covar_samp(1, 1) from analytics group by aa, bb;
select covar_pop(1, 1) from analytics group by aa, bb;


select covar_samp(NULL, aa) from analytics group by aa;
select covar_pop(NULL, aa) from analytics group by aa;
select covar_samp(NULL, aa) from analytics group by bb;
select covar_pop(NULL, aa) from analytics group by bb;
select covar_samp(NULL, aa) from analytics group by aa, bb;
select covar_pop(NULL, aa) from analytics group by aa, bb;

select covar_samp(aa, NULL) from analytics group by aa;
select covar_pop(aa, NULL) from analytics group by aa;
select covar_samp(aa, NULL) from analytics group by bb;
select covar_pop(aa, NULL) from analytics group by bb;
select covar_samp(aa, NULL) from analytics group by aa, bb;
select covar_pop(aa, NULL) from analytics group by aa, bb;

select covar_samp(NULL, NULL) from analytics group by aa;
select covar_pop(NULL, NULL) from analytics group by aa;
select covar_samp(NULL, NULL) from analytics group by bb;
select covar_pop(NULL, NULL) from analytics group by bb;
select covar_samp(NULL, NULL) from analytics group by aa, bb;
select covar_pop(NULL, NULL) from analytics group by aa, bb;

rollback;
