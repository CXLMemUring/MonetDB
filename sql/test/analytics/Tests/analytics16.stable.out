stdout of test 'analytics16` in directory 'sql/test/analytics` itself:


# 18:06:21 >  
# 18:06:21 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34861" "--set" "mapi_usock=/var/tmp/mtest-290531/.s.monetdb.34861" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-statistics-analytics/BUILD/var/MonetDB/mTests_sql_test_analytics" "--set" "embedded_c=true"
# 18:06:21 >  

# MonetDB 5 server v11.36.0 (hg id: 3ca5b1c53516+cc1d71a81796+)
# This is an unreleased version
# Serving database 'mTests_sql_test_analytics', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 128bit integers
# Found 15.498 GiB available main-memory of which we use 12.631 GiB
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2020 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.localdomain:34861/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-290531/.s.monetdb.34861
# MonetDB/GIS module loaded
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_querylog.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_temporal.sql
# loading sql script: 18_index.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_views.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 25_debug.sql
# loading sql script: 26_sysmon.sql
# loading sql script: 27_rejects.sql
# loading sql script: 39_analytics.sql
# loading sql script: 39_analytics_hge.sql
# loading sql script: 40_geom.sql
# loading sql script: 40_json.sql
# loading sql script: 40_json_hge.sql
# loading sql script: 41_md5sum.sql
# loading sql script: 45_uuid.sql
# loading sql script: 46_profiler.sql
# loading sql script: 51_sys_schema_extension.sql
# loading sql script: 58_hot_snapshot.sql
# loading sql script: 60_wlcr.sql
# loading sql script: 61_wlcr.sql
# loading sql script: 72_fits.sql
# loading sql script: 74_netcdf.sql
# loading sql script: 75_lidar.sql
# loading sql script: 75_shp.sql
# loading sql script: 75_storagemodel.sql
# loading sql script: 80_statistics.sql
# loading sql script: 80_udf.sql
# loading sql script: 80_udf_hge.sql
# loading sql script: 81_tracer.sql
# loading sql script: 85_bam.sql
# loading sql script: 90_generator.sql
# loading sql script: 90_generator_hge.sql
# loading sql script: 99_system.sql
# MonetDB/SQL module loaded

# 18:06:22 >  
# 18:06:22 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-290531" "--port=34861"
# 18:06:22 >  

#start transaction;
#create table analytics (aa int, bb int);
#insert into analytics values (15, 3), (3, 1), (2, 1), (5, 3), (NULL, 2), (3, 2), (4, 1), (6, 3), (8, 2), (NULL, 4);
[ 10	]
#select covar_samp(aa, aa) from analytics;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 17.64285714	]
#select covar_samp(bb, bb) from analytics;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 1.066666667	]
#select covar_samp(aa, bb) from analytics;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 2.428571429	]
#select covar_samp(bb, aa) from analytics;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 2.428571429	]
#select covar_pop(aa, aa) from analytics;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 15.4375	]
#select covar_pop(bb, bb) from analytics;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0.96	]
#select covar_pop(aa, bb) from analytics;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 2.125	]
#select covar_pop(bb, aa) from analytics;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 2.125	]
#select covar_samp(aa, aa) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ 0	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(bb, bb) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ 0.5	]
[ NULL	]
[ NULL	]
[ 2	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(aa, bb) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ 0	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(bb, aa) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ 0	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(aa, aa) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
[ 0	]
[ 0	]
[ 0	]
#select covar_pop(bb, bb) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0.25	]
[ 0	]
[ 0	]
[ 1	]
[ 0	]
[ 0	]
[ 0	]
#select covar_pop(aa, bb) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
[ 0	]
[ 0	]
[ 0	]
#select covar_pop(bb, aa) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
[ 0	]
[ 0	]
[ 0	]
#select covar_samp(aa, aa) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 30.33333333	]
[ 1	]
[ 12.5	]
[ NULL	]
#select covar_samp(bb, bb) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_samp(aa, bb) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_samp(bb, aa) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_pop(aa, aa) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 20.22222222	]
[ 0.6666666667	]
[ 6.25	]
[ NULL	]
#select covar_pop(bb, bb) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
#select covar_pop(aa, bb) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_pop(bb, aa) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_samp(aa, aa) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(bb, bb) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(aa, bb) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(bb, aa) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(aa, aa) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_pop(bb, bb) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
#select covar_pop(aa, bb) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_pop(bb, aa) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_samp(aa, 1) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ 0	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(aa, 1) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
[ 0	]
[ 0	]
[ 0	]
#select covar_samp(aa, 1) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_pop(aa, 1) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_samp(aa, 1) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(bb, 1) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
#select covar_samp(1, aa) from analytics group by aa;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ 0	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(1, aa) from analytics group by aa;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
[ 0	]
[ 0	]
[ 0	]
#select covar_samp(1, aa) from analytics group by bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_pop(1, aa) from analytics group by bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_samp(1, aa) from analytics group by aa, bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(1, aa) from analytics group by aa, bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_samp(1, 1) from analytics group by aa;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ 0	]
[ NULL	]
[ NULL	]
[ 0	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(1, 1) from analytics group by aa;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
#select covar_samp(1, 1) from analytics group by bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ NULL	]
#select covar_pop(1, 1) from analytics group by bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
#select covar_samp(1, 1) from analytics group by aa, bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(1, 1) from analytics group by aa, bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
[ 0	]
#select covar_samp(NULL, aa) from analytics group by aa;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(NULL, aa) from analytics group by aa;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(NULL, aa) from analytics group by bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(NULL, aa) from analytics group by bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(NULL, aa) from analytics group by aa, bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(NULL, aa) from analytics group by aa, bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(aa, NULL) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(aa, NULL) from analytics group by aa;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(aa, NULL) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(aa, NULL) from analytics group by bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(aa, NULL) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(aa, NULL) from analytics group by aa, bb;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(NULL, NULL) from analytics group by aa;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(NULL, NULL) from analytics group by aa;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(NULL, NULL) from analytics group by bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(NULL, NULL) from analytics group by bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_samp(NULL, NULL) from analytics group by aa, bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#select covar_pop(NULL, NULL) from analytics group by aa, bb;
% .%1 # table_name
% %1 # name
% double # type
% 24 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
[ NULL	]
#rollback;

# 18:06:22 >  
# 18:06:22 >  "Done."
# 18:06:22 >  

